Gemfile.lock

```diff
@@ -99,6 +103,8 @@ GEM
     honeybadger (1.16.7)
       json
     i18n (0.7.0)
+    jmespath (1.0.2)
+      multi_json (~> 1.0)
```

Comment:
:+1: overall but this dependency ties us tighter to `multi_json`. I’m okay with that but it will make your project to completely eliminate the `multi_json` dependency more difficult.